doc/articles/get-started-wizard.md

```diff
@@ -10,31 +8,40 @@
 * Next you can try our [Bug Tracker sample walkthrough](https://platform.uno/docs/articles/getting-started-tutorial-2.html)
 * More advanced examples in our [Uno.Samples repository](https://github.com/unoplatform/uno.samples)
 * Fork a fully-fledged [Ch9 application and source code ](https://platform.uno/code-samples/#ch9)
-* You can find detailed documentation on Uno topics [on our web site](https://platform.uno/docs/articles/intro.html).
-* Run the [uno-check command-line tool](external/uno.check/doc/using-uno-check.md) to ensure your dev environment is set up correctly
+* You can find detailed resources for Getting Started with Uno [on our web site](https://platform.uno/blog/uno-platform-learning-resources-2023-update/).
 
 ## Common Issues
 The Uno Platform features and support are constantly evolving, but you may encounter some of the issues below while building your application. Some additional topics may be found in the [GitHub top discussions](https://github.com/unoplatform/uno/discussions).
 
-#### WebAssembly: Access to fetch at 'https://XXXX' from origin 'http://XXXX' has been blocked by CORS policy
+#### 1. The XAML editor shows `The type 'page' does not support direct content` message
+
+XAML Intellisense is not working properly in Visual Studio when the active project is not the UWP one.
+
+To work around this issue, close all XAML editors, open a C# file and select the '[MyApp].Windows' in the top-left drop-down list of the text editor sector. Once selected, re-open the XAML file.
+
+#### 2. XAML Hot Reload troubleshooting
+
+The XAML Hot reload provides a Visual Studio for Windows output window name "Uno Platform" with diagnostics messages. You can find additional information there in case XAML Hot Reload does not work properly.
```

Comment:
```suggestion
The XAML Hot reload provides a Visual Studio for Windows output window named "Uno Platform" with diagnostics messages. You can find additional information there in case XAML Hot Reload does not work properly.
```